index.js

```diff
@@ -45,8 +45,12 @@ const template = [
     submenu: [
       {
         label: 'Learn More',
-        click () { require('electron').shell.openExternal('https://cabal.chat') }
-      }
+        click () { require('electron').shell.openExternal('https://cabal-club.github.io/') }
```

Comment:
Does this need to change?

Reply:
Hmm, looks like an SSL issue now that I look at it closer. Trying to load https://cabal.chat just stalls in Chrome, but simply `cabal.chat` redirects just fine to https://cabal-club.github.io/

I'll patch this then.

Reply:
resolved with 2700c3d